foursight_core/run_result.py

```diff
@@ -243,7 +260,7 @@ def wrapper(obj):
                 return self.filename_to_datetime(filename), total
 
             if after_date is not None:
-                history = list(filter(lambda k: wrapper(k) >= after_date, history))
+                history = list(filter(lambda k: wrapper(k)[0] >= after_date, history))
```

Comment:
I hope this object is always in proper form. Maybe it's my C++ experience, but I get nervous when I see chained `[ ]`'s.

Reply:
This is in response to a change to return value David made, which I generally think is fine. It is tested so hopefully is fine...